packages/utilities/etc/utilities.api.md

```diff
@@ -64,10 +64,10 @@ export class Async {
     setImmediate(callback: () => void, targetElement?: Element | null): number;
     setInterval(callback: () => void, duration: number): number;
     setTimeout(callback: () => void, duration: number): number;
-    throttle<T extends Function>(func: T, wait?: number, options?: {
+    throttle<T extends (...args: any[]) => any>(func: T, wait?: number, options?: {
         leading?: boolean;
         trailing?: boolean;
-    }): T | (() => void);
+    }): T;

```

Comment:
Are there any tests for this? I'd be interested in potentially adding tests that catch the compilation issue I ran across.